__init__.py

```diff
@@ -10,19 +10,196 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+
+import requests
+import simplematch
+from ovos_classifiers.heuristics.keyword_extraction import HeuristicExtractor
+from ovos_plugin_manager.templates.solvers import QuestionSolver
+from ovos_utils import classproperty
 from ovos_utils.gui import can_use_gui
-from adapt.intent import IntentBuilder
-from mycroft.skills.common_query_skill import CommonQuerySkill, CQSMatchLevel
-from mycroft.skills.core import intent_handler
-from neon_solver_ddg_plugin import DDGSolver
+from ovos_utils.intents import IntentBuilder
 from ovos_utils.process_utils import RuntimeRequirements
-from ovos_utils import classproperty
+from ovos_workshop.decorators import intent_handler
+from ovos_workshop.skills.common_query_skill import CommonQuerySkill, CQSMatchLevel
+
+
+class DDGIntents:
+
+    # TODO use padacioso and allow localization with .intent files
+
+    @staticmethod
+    def match(query, lang):
+        if not lang.startswith("en"):
```

Comment:
Does this mean that you can only use this skill with English right now? Any other setting will simply return None?

Reply:
DDG api only allows english input, the reason we use a solver plugin is to have automatic translation of queries back and forth

that check is only there really to allow importing that class and using it standalone and such, in the skill context we only handle english input